recipes/c-blosc/all/conanfile.py

```diff
@@ -109,6 +109,13 @@ def generate(self):
     def _patch_sources(self):
         apply_conandata_patches(self)
         rmdir(self, os.path.join(self.source_folder, "cmake"))
+        # Do not install system libs
+        replace_in_file(
+            self,
+            os.path.join(self.source_folder, "CMakeLists.txt"),
+            "include(InstallRequiredSystemLibraries)",
```

Comment:
Can you disable this by setting the CMake variable `CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP` to true? Docs [here](https://cmake.org/cmake/help/latest/module/InstallRequiredSystemLibraries.html#installrequiredsystemlibraries).

Reply:
I guess yes